src/core/ticker/Ticker.js

```diff
@@ -107,34 +107,31 @@ class Ticker
          */
         this.started = false;
 
-        // Bind tick
-        this._tick = this._tick.bind(this);
-    }
+        /**
+         * Internal tick method bound to ticker instance.
+         * This is because in early 2015, Function.bind
+         * is still 60% slower in high performance scenarios.
+         * Also separating frame requests from update method
+         * so listeners may be called at any time and with
+         * any animation API, just invoke ticker.update(time).
+         *
+         * @private
+         */
+        this._tick = (time) => {
```

Comment:
Babel does not transpile fat arrow functions into binds, so this is, in fact not the same as Function.bind. Here's how it get's [transpiled](http://s3-eu-west-1.amazonaws.com/pixi.js/dev-es6/pixi.js), which is the same as currently on dev.

```js
var _this = this;
this._tick = function (time) {

    _this._requestId = null;

    if (_this.started) {
        // Invoke listeners now
        _this.update(time);
        // Listener side effects may have modified ticker state.
        if (_this.started && _this._requestId === null && _this._emitter.listeners(TICK, true)) {
            _this._requestId = requestAnimationFrame(_this._tick);
        }
    }
};
```